src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -3877,7 +3894,7 @@ void processResource(IOperation resource, ArrayBuilder<(IVariableDeclarationOper
             }
         }
 
-        private void AddDisposingFinally(IOperation resource, bool knownToImplementIDisposable, ITypeSymbol iDisposable, bool isAsynchronous)
+        private void AddDisposingFinally(IOperation resource, IMethodSymbol? disposeMethod, bool isAsynchronous, bool requiresRuntimeConversion = false)
```

Comment:
>requiresRuntimeConversion [](start = 118, length = 25)

Please make it a required parameter. Its value is significant and everybody (a dev and reviewers) should pay special attention to it.